hrjobcontract/js/src/job-contract/services/utils.js

```diff
@@ -118,7 +118,8 @@ define([
                 var deffered = $q.defer();
 
                 API.resource('HRPayScale','get',{
-                    sequential: 1
+                    sequential: 1,
+                    is_active: 1
```

Comment:
mmmm I am not really sure about this .. this method only make an ajax call to get some data so I Am not sure how it could be tested ... at the end , the ajax call will operate on the database you are using and not on test database .